docs/whats-new.md

```diff
@@ -2,6 +2,23 @@
 
 This page contains highlights of each deck.gl release. Also check our [vis.gl blog](https://medium.com/vis-gl) for news about new releases and features in deck.gl.
 
+## deck.gl v8.5 (In development)
+
+### MVT Layer
+
+#### Direct binary parsing
+
+Mapbox Vector Tiles parsing throughput is now 2-3x faster, due to MVT tiles being parsed directly into binary attributes rather than GeoJSON, and additional work (including [triangulation](https://github.com/visgl/loaders.gl/blob/master/docs/whats-new.md#v30-in-development))being performed on worker threads. Speed comparison on some example data sets (MVT tiles parsed per second):
+
+|  Data set                 | `binary: false` | `binary: true` | Speed increase |
+| ------------------ | ----------- | ------ | -------------- |
+| Block groups       | 2.86/s      | 5.57/s | 1.94x          |
+| Census layer       | 6.09/s      | 11.9/s | 1.95x          |
```

Comment:
```suggestion
| Block Groups       | 2.86/s      | 5.57/s | 1.94x          |
| Census Layer       | 6.09/s      | 11.9/s | 1.95x          |
```
for consistent capitalization across lines

Reply:
Thanks for the corrections, Kyle :)